frontend/src/wrapper/ModelSearchField.tsx

```diff
@@ -2,44 +2,31 @@ import SearchIcon from '@mui/icons-material/Search'
 import { Box, InputBase, List, ListItemButton, ListItemText, Popover, Stack } from '@mui/material'
 import { alpha, styled } from '@mui/material/styles'
 import { useListModels } from 'actions/model'
-import { ChangeEvent, useState } from 'react'
+import { ChangeEvent, useMemo, useState } from 'react'
+import Loading from 'src/common/Loading'
 import Link from 'src/Link'
 import MessageAlert from 'src/MessageAlert'
 import useDebounce from 'utils/hooks/useDebounce'
 
 const Search = styled('div')(({ theme }) => ({
-  position: 'relative',
   borderRadius: theme.shape.borderRadius,
   backgroundColor: alpha(theme.palette.common.white, 0.15),
   '&:hover': {
     backgroundColor: alpha(theme.palette.common.white, 0.25),
   },
-  marginLeft: 0,
   width: '100%',
   [theme.breakpoints.up('sm')]: {
     marginLeft: theme.spacing(1),
     width: 'auto',
   },
 }))
 
-const SearchIconWrapper = styled('div')(({ theme }) => ({
-  padding: theme.spacing(0, 2),
-  height: '100%',
-  position: 'absolute',
-  pointerEvents: 'none',
-  display: 'flex',
-  alignItems: 'center',
-  justifyContent: 'center',
-}))
-
 const StyledInputBase = styled(InputBase)(({ theme }) => ({
   color: 'inherit',
   width: '100%',
-  paddingRight: 4,
+  paddingRight: theme.spacing(4),
```

Comment:
Our spacing is in increments of 8px, so this is a huge amount of padding. `theme.spacing(1)` will probably look right. More than it was before, but we rarely use such as small amount of padding as 4px